api/api.proto

```diff
@@ -307,6 +307,8 @@ message VolumeSpec {
   bool journal = 25;
   // Sharedv4 is true if this volume can be accessed via sharedv4.
   bool sharedv4 = 26;
+  // FsResizeRequired if an FS resize is required on the volume.
+  bool fs_resize_required = 27;
```

Comment:
Can this value be set in Volume.Set(). If it can, then this value needs to be added to VolumeSpecUpdate and to the SDK accordingly. If not, then just add a comment on this value saying that this value cannot be set on a volume after creation.

Reply:
Doesn't need to be in the SDK. There is no enforcement for the flag here, it depends on the implementation, so a comment here, either way, would be wrong.

Reply:
There is nothing preventing it from being changed. Some implementations could allow it to be changed. There shouldn't be a comment here for that.

Reply:
Ok, so if it can be changed, then it needs to add it to VolumeSpecUpdate and SDK server handler. Then the driver can decide if it can be changed.